docs/src/rules/global-strict.md

```diff
@@ -1,8 +1,13 @@
-# global-strict
+---
+title: global-strict
+layout: doc
+edit_link: https://github.com/eslint/eslint/edit/main/docs/src/rules/global-strict.md
+
```

Comment:
To confirm, this has a blank line instead of `rule_type` because the rule was removed and replaced?

Reply:
Correct.